src/components/FilterDropdown.jsx

```diff
@@ -43,13 +44,15 @@ function FilterDropdown(props) {
 
 FilterDropdown.propTypes = {
   label: PropTypes.string.isRequired,
-  defaultSelect: PropTypes.string,
+  defaultValue: PropTypes.string,
   items: PropTypes.objectOf(PropTypes.string).isRequired,
   removeNone: PropTypes.bool,
+  onChange: PropTypes.func,
 };
 
 FilterDropdown.defaultProps = {
-  defaultSelect: '',
+  defaultValue: '',
+  onChange: null,
```

Comment:
Rather than defaulting to `null` I think it should be defaulted to a no-op function like `(e) => {}`. Then you don't have to check if it's null.

Reply:
I think the e in not needed right? I pushed my changes to this and removed the checking of `null`

Reply:
I don't think it is. If the linter complains just remove it.